NOTES.UNIX

```diff
@@ -89,3 +90,28 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         '-Wl,--enable-new-dtags,-rpath,$(LIBRPATH)'
+
+ It might be worth noting that some/most ELF systems implement support
+ for runtime search path relative to the directory containing current
+ executable, by interpreting $ORIGIN along with some other internal
+ variables. Consult your system documentation.
+
+ Linking your application
+ ------------------------
+
+ Third-party applications dynamically linked with OpenSSL (or any other)
+ shared library face exactly same problem with non-default locations.
+ Abovementioned OpenSSL config options might or might not have bearing
```

Comment:
s/Abovementioned OpenSSL config options/The OpenSSL config options mentioned above/